MAUI/Cartesian-Charts/Tooltip.md

```diff
@@ -246,6 +246,46 @@ The [Show]() method is used to activate the tooltip at the specified location.
 
 {% endhighlight %}
 
+{% highlight C# %}
+
+    Stacklayout layout = new Stacklayout();
+    
+    SfCartesianChart chart = new SfCartesianChart();
+    ....
+    ChartTooltipBehavior tooltipBehavior = new ChartTooltipBehavior();
+    chart.TooltipBehavior = tooltipBehavior;
+    ...
+    ColumnSeries series1 = new ColumnSeries()
```

Comment:
@aruljenithberkmans 
No need to series code, remove it

Reply:
i removed it